examples/cli-only-template/src/linkedin-data-extraction.cli.ts

```diff
@@ -27,9 +51,13 @@ async function cli() {
   let sessionAndWindow = undefined;
 
   try {
-    sessionAndWindow = await service.initializeSessionAndBrowser(profileId);
+    sessionAndWindow = await service.initializeSessionAndBrowser(profileAnswer === "use" ? profileName : undefined);
     const { session, windowInfo } = sessionAndWindow;
 
+    if (profileAnswer === "create") {
```

Comment:
Should this be both? Wouldn't we also want to save changes to an existing profile if they chose "use"?
```suggestion
    if (profileAnswer === "create" || profileAnswer === "use") {
```

Reply:
nice catch, let's add it